sparkmagic/sparkmagic/livyclientlib/command.py

```diff
@@ -65,7 +68,14 @@ def _get_statement_output(self, session, statement_id):
                     return (True, u"")
 
                 if statement_output[u"status"] == u"ok":
-                    return (True, statement_output[u"data"][u"text/plain"])
+                    data = statement_output[u"data"]
+                    png_encoded = data.get("image/png")
+                    if png_encoded:
+                        image = Image(base64.b64decode(png_encoded))
+                        return (True, image)
+                    else:
```

Comment:
IPython.display.Image should also support png/gif?

Reply:
Yeah, probably, but Livy specifically only returns PNG at the moment.